src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -109,9 +109,19 @@
     @Option(doc = "Read two barcode SAM tag (ex. BX for 10X Genomics)", optional = true)
     public String READ_TWO_BARCODE_TAG = null;
 
+    @Option(doc = "If true remove optical duplicates from the output even if REMOVE_DUPLICATES is false.")
+    public boolean REMOVE_OPTICAL_DUPLICATES = false;
+
+    @Option(doc = "If set write a tag with this name and OPTICAL_DUPLICATE_TAG_VALUE on to each read that is an optical duplicate.", optional=true)
+    public String OPTICAL_DUPLICATE_TAG_NAME = null;
```

Comment:
according to https://github.com/samtools/hts-specs/issues/121 this should be hard-coded to DT, a new tag to be added to SAMUtils.